src/main/twirl/uk/gov/hmrc/sca/views/ScaLayout.scala.html

```diff
@@ -43,10 +45,12 @@
         hmrcTimeoutDialogHelper: HmrcTimeoutDialogHelper,
         hmrcAccountMenu: HmrcAccountMenu,
         ptaScripts: PtaScripts,
+        ptaHead: PtaHead,
```

Comment:
Question, is PtaHead providing CSS for the attorney banner and the banner doesn't work without this?

Reply:
we need the js and css from pta, that's why this is required here.